cockroachcloud/export-logs.md

```diff
@@ -330,7 +330,7 @@ Perform the following steps to enable log export from your {{ site.data.products
                      },
                      {
                          log_name: "devops",
-                         channels: ["OPS", "HEALTH", "STORAGE"]
+                         channels: ["OPS", "HEALTH", "STORAGE"],
                          min_level: "WARNING"
                      },
```

Comment:
same as above. keys should be quoted.

Reply:
Also quote the value `GCP_CLOUD_LOGGING` but not the value `true`.